layouts/solo/iframe-static.html

```diff
@@ -79,7 +82,24 @@
 
       return settings;
     }());
+    var playlist = settings.playlist || null;
+    if (playlist && "show" === playlist) {
+      var menu = new Menu(player, channelVideos);
 
+      player.onPlayerReady = function(){
```

Comment:
@karloscastaneda so this didn't worked, callbacks where not appearing in the constructor options